cartographer/cloud/internal/local_trajectory_uploader.cc

```diff
@@ -183,6 +183,14 @@ void LocalTrajectoryUploader::TryRecovery() {
     }
   }
 
+  // Because the trajectories may be interrupted on the uplink side, we can no
```

Comment:
What do you mean by "interrupted" here?

Reply:
Interrupted means that the uplink does not have all sensor data (because the connection was interrupted)